packages/config/src/utils.ts

```diff
@@ -0,0 +1,17 @@
+export const deepSearch = (
```

Comment:
Add a jsdoc comment with an explanation of what the method is doing and it's params.